src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> head;
+    private Node<T> tail;
+
+    private class Node<T> {
+        private T value;
+        private Node<T> prev;
+        private Node<T> next;
+
+        Node(Node<T> prev, T x, Node<T> next) {
+            this.prev = prev;
+            value = x;
```

Comment:
use this

Reply:
@QQBratka  we should use this only if the name of parameters are the same, in this code, we have different variable names, so we can skip `this` keyword